jenkins-genomel.planx-pla.net/portal/gitops.json

```diff
@@ -2,31 +2,8 @@
   "gaTrackingId": "UA-119127212-12",
   "graphql": {
     "boardCounts": [
```

Comment:
why are we removing these?

Reply:
graphql throws an error while loading the portal page.